src/current/cockroachcloud/upgrade-to-v23.2.md

```diff
@@ -9,6 +9,9 @@ pre_production_preview: false
 pre_production_preview_version: v23.2.0-rc.2
 ---
 
+{% capture previous_version_numeric %}{{ page.prev_version | strip_first: 'v'}}{% endcapture %}
```

Comment:
Maybe this will fix the display issues?

```suggestion
{% capture previous_version_numeric %}{{ page.prev_version | remove_first: 'v'}}{% endcapture %}
```

Reply:
How silly! Thank you.